l10n_de_tax_statement/__manifest__.py

```diff
@@ -11,6 +11,7 @@
     'website': 'https://github.com/openbig/l10n-germany',
     'depends': [
         'account_tax_balance',
+        'l10n_de',
```

Comment:
So at the end the dependency on `l10n_de` was added :smile: 
If you make this module dependent on `l10n_de`, the configuration mechanism can be simplified (especially the wizard). On the other side, by doing this way, you are forced to install `l10n_de` even if you don't use it.

Reply:
This is why I suggested a glue module (dependency to l10n_de and l10n_de_tax_statement with auto_install = True). It will get automatically installed if both of the dependency is fulfilled. In that case you don't have to add any checks because Odoo is doing it for you.

Reply:
@fkantelberg did you read my https://github.com/OCA/l10n-germany/pull/38#discussion_r263787940 ?

Reply:
Yes but I don't see any reason to implement tests which are already in odoo if you use the features like autoinstall in modules. But I just want to help to make the module more readable and usable.

Reply:
@astirpe @fkantelberg I will now test locally what happens, if i remove the l10n_de dependency in order to simulate the scenario c.) i mentioned above, as for sure that could theoretically a user descicion, even if it is near 0% that it really would happen.
If that will work as expected i will remove the l10n_de dependency again.